cypress/integration/onsite/onsite_group_landing_page_spec.js

```diff
@@ -11,12 +11,32 @@ describe('Given I navigate to /Onsite Group Page:', function () {
         
       cy.ignoreMatchingErrors(errorsToIgnore);
       cy.visit('/groups/onsite');
-       
-    });
+      });
  
+    it ('Onsite Group card for Financial Peace should be last', function(){
+      cy.get('ul').as('onsiteGroupCards');
+      cy.get('@onsiteGroupCards').should('have.length', onsiteGroupList.length);
+      const financialPeaceIndex = onsiteGroupList.length;
+      cy.get('@onsiteGroupCards').eq(financialPeaceIndex-1).find('a').should('have.attr', 'href', '/groups/onsite/financial-peace/uptown'); 
+    });
+
     it ('Test Onsite Group', function(){
-      cy.get('h3').as('onsiteCards');
-      cy.get('@onsiteCards').should('have.length', onsiteGroupList.length);
+      cy.get('h3').as('onsiteGroupCards');
+      cy.get('@onsiteGroupCards').should('have.length', onsiteGroupList.length);
     });
 
-});
\ No newline at end of file
+    [0,1,2].forEach(index => {
+        before(function () {
+          
+      });
+   
+      it(`Onsite card #${index} should have a Title`, function () {
+        let onsite;
+        let title;
+        onsite = onsiteGroupList[index];
+        title = onsite.title.text;
```

Comment:
Why are the variable declarations and assignments split?

Reply:
no specific reason, they are combined now